### DISCUSSION TITLE
Publish highspy 1.6.0 wheels

### DISCUSSION THREAD

**Discussion by AdeelK93** [ERGO-Code/HiGHS#1441] [General]
I believe 1.6.0 includes the fix for wheel build issues (https://github.com/ERGO-Code/HiGHS/pull/1343) and allowing macos to install highspy via pypi rather than building from source. 

It would be great if the 1.6.0 wheels could be built and uploaded! Thank you!!

**Comment by jajhall** [comment]
Thanks, but we need to be confident that this allows highspy to be installed via PyPI on all architectures

**Comment by AdeelK93** [comment]
anything i can do to help with that? would a GHA testing matrix help?

**Comment by galabovaa** [comment]
Thank you for your offer, the GHA matrix is not the issue. Do you know much about meson? One issue at the moment is the shared library lookup. 

https://github.com/ERGO-Code/HiGHS/actions/runs/6403243988/job/17381362923

**Reply by AdeelK93** [reply]
No unfortunately ☹️ sorry

**Reply by metab0t** [reply]
Upstream fix for meson-python has landed: https://github.com/mesonbuild/meson-python/pull/510
Maybe we can use meson-python#master in CI to build the wheel?

**Comment by galabovaa** [comment]
Moving this to a discussion, updating highspy is a WIP

**Reply by metab0t** [reply]
@galabovaa Is it time to reenable build-wheels and test-python-api workflows?

**Reply by AdeelK93** [reply]
Does @metab0t's contribution allow us to try building wheels again? Thanks!

**Reply by AdeelK93** [reply]
Hi I wanted to check in on this - do you think we can reenable build-wheels and test out if we can publish new wheels? Thank you!

**Reply by jajhall** [reply]
@galabovaa is working on it

**Comment by AdeelK93** [comment]
Do you think wheels for Python 3.12 can also be included as a part of this project? Thank you!

**Comment by dthemg** [comment]
Any updates on this? Some of the bug-fixes would be much appreciated in my team 🙏

**Reply by jajhall** [reply]
@galabovaa is making progress, and `highspy` can be installed locally from compiled HiGHS source code

**Comment by jajhall** [comment]
This has now been achieved with https://pypi.org/project/highspy/1.7.1.dev1/

**Reply by AdeelK93** [reply]
Thank you! Why is this considered dev and not stable? Anything to watch out for?

**Reply by jajhall** [reply]
No, we just wanted to see whether anyone reported any PyPI problems - and none has been raised

There are a couple of issues to fix resulting from the release of v1.7.0, and then we'll put out v1.7.1
